### ISSUE TITLE
Multiple Events Bug

### ISSUE THREAD

**Issue by lpcoding** [ctsit/form_render_skip_logic#55]
If you place a condition in an instrument inside a multiple event, the condition is not true only in the current event instance but in all instance of the repeated event.

![FireShot Capture 029 - Classic Database - REDCap - redcap gimema it](https://user-images.githubusercontent.com/15016704/64417692-e01bc200-d099-11e9-8f2f-ac94289bbc72.png)

In this example i set that if the baseline data in incomplete it should disable the three month intrument. 
The second instance should be visible according to the rule

**Comment by pbchase** [comment]
Can you provide an XML export of the REDCap project you tested with and a screenshot of the FRSL configuration for that project?

**Comment by pbchase** [comment]
I don't think we ever added explicit support for repeating events in this module, but neither do I see this documented in the README. So this is probably a documentation bug in the short term and feature enhancement in the long term.

**Comment by pbchase** [comment]
Release 3.3.10 addresses the documentation issue. So that addresses the bug aspect, but not the feature enhancement.

**Comment by aandresalvarez** [comment]
Hi @pbchase , I am just wondering if you have plans to add the Repeating event support functionality in the upcoming months?
